wiki/How_You_Can_Help!/en.md

```diff
@@ -1,77 +1,77 @@
 
 ## Contribute to the future of osu!
 
-Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which you see today.
+Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which is what you see today! <!-- I personally don't like this line: it's oddly short and doesn't really finish what it started. But I don't really know what else to put here. Maybe a link to a page explaining what osu!lazer is at the very least. If any of you guys have ideas, please tell me. -toaster -->
 
-You can help make them even better, and make us closer to the future of osu! by helping on these areas:
+If you would like to help make these projects even better and bring us closer to the future of osu!, then please consider…
 
-- Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is much to implement on the new version of osu!, and you can help make it happen!
+- Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is so much to implement on the new version of osu!, and you can help make it happen!
 - Make the [new website](https://github.com/ppy/osu-web) even better.
 
 ## Avoid being negative! Make people happy!
 
-There is an old saying that you should become part of the solution, not the problem. You can help on osu! by making the community a brighter place. People complain, but there are many ways you can help lighten up the mood in the community:
+There is an old saying that you should become part of the solution, not the problem. Simply by following this old saying, you'd be helping osu! by making the community a brighter place. People will always complain, but there are still so many ways you can help lighten up the mood in the community:
 
-- Be friendly to each other. Life is too short to argue.
-- [Report cheaters when you see them](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play). Do not complain about them on other mediums, a simple `!report <username> seems to be cheating` can fix your problems within minutes to hours.
-- Go out of your way to mediate negative situations and turn them into positive outcomes.
+- Be friendly to each other; life is too short to argue.
+- [Report cheaters when you see them](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play). Please do not complain about them through other mediums; a simple `!report <username> seems to be cheating` can fix your problems much quicker.
+- Go out-of-your-way to mediate negative situations and turn them into positive outcomes.
 
-While this article cannot cover how much more you can help the osu! community, simply playing the game as well can make osu! even more awesome!
+While this article cannot cover how much more you can help the osu! community, simply playing the game as well can make osu! even more awesome! Good luck out there!
```

Comment:
This looks rather unnecessary.

Reply:
Just a little cherry-on-top. Personally, I think it's nice, but if people don't like it, I'll remove it.

Reply:
The concluding sentence is enough as it is IIRC